openmc/summary.py

```diff
@@ -520,12 +521,18 @@ def _read_tallies(self):
             # Create Tally object and assign basic properties
             tally = openmc.Tally(tally_id, tally_name)
 
+            # Read scattering moment order strings (e.g., P3, Y-1,2, etc.)
```

Comment:
I realize this is a copy-and-paste error, but it should be "Y1,2", not "Y-1,2". Can you fix it here and in statepoint.py too?